src/app/common/services/bootstrap/bootstrap.service.ts

```diff
@@ -18,6 +18,41 @@ export class BootstrapService {
     'http_port': '9000',
     'mqtt_url': 'localhost:1883',
     'edgex_url': 'http://localhost:48090/api/v1/',
+    'nats_url': 'localhost:4222',
+    'export_config': {
+      'exp': {
+        'log_level': 'debug',
+        'nats': 'nats://localhost:4222',
+        'port': '8170',
+      },
+      'mqtt': {
+        'ca_path': 'ca.crt',
+        'cert_path': 'thing.crt',
+        'channel': '35860baf-6660-4f47-a5c5-8eb1bd176175',
+        'host': 'tcp://84.201.171.65:1883',
+        'mtls': false,
+        'password': 'e9c03586-7422-4772-8631-9e0d3f152ca1',
+        'priv_key_path': 'thing.key',
+        'qos': 0,
+        'retain': false,
+        'skip_tls_ver': false,
+        'username': 'b9688074-9bc1-4736-8132-7fd45a0736ef',
+      },
+      'routes': [
+        {
+          'mqtt_topic': 'channels/35860baf-6660-4f47-a5c5-8eb1bd176175/messages',
+          'nats_topic': 'adc.samples',
+          'subtopic': '',
+          'type': 'plain',
+        },
+        {
+          'mqtt_topic': 'channels/35860baf-6660-4f47-a5c5-8eb1bd176175/messages',
+          'nats_topic': 'telegraf',
+          'subtopic': '',
+          'type': 'plain',
+        },
+      ],
+    },
```

Comment:
Why do we need to use literals for the configuration. If the properties of this object will be filled out later, please use zero values. If we need to fill in the values, please use sensible defaults and make a form.

Reply:
Done